modules/data/src/main/java/io/fluo/webindex/data/fluo/DomainExport.java

```diff
@@ -14,30 +14,29 @@
 
 package io.fluo.webindex.data.fluo;
 
-import java.util.Collection;
 import java.util.Collections;
+import java.util.Map;
 
-import io.fluo.webindex.core.Constants;
-import io.fluo.webindex.data.recipes.Transmutable;
-import org.apache.accumulo.core.data.Mutation;
+import io.fluo.api.data.Bytes;
+import io.fluo.api.data.RowColumn;
+import io.fluo.recipes.accumulo.export.DifferenceExport;
 
-public class DomainExport implements Transmutable<String> {
-  private long count;
+import static io.fluo.webindex.data.util.FluoConstants.PAGECOUNT_COL;
+
+public class DomainExport extends DifferenceExport<String, Long> {
 
   public DomainExport() {}
 
-  public DomainExport(long c) {
-    this.count = c;
+  public DomainExport(Long oldCount, Long newCount) {
+    super(oldCount, newCount);
   }
 
   @Override
-  public Collection<Mutation> toMutations(String domain, long seq) {
-    Mutation m = new Mutation("d:" + domain);
+  protected Map<RowColumn, Bytes> generateData(String domain, Long count) {
     if (count == 0) {
-      m.putDelete(Constants.DOMAIN, Constants.PAGECOUNT, seq);
-    } else {
-      m.put(Constants.DOMAIN, Constants.PAGECOUNT, seq, count + "");
+      return Collections.emptyMap();
```

Comment:
had to look at this a second to ensure it deletes when new value is zero... I think it does